_data/cryptocurrencies.yml

```
@@ -406,6 +406,14 @@ websites:
       - totp
     doc: https://support.hitbtc.com/en/support/solutions/articles/63000224976
 
+  - name: HUOBI
+    url: https://www.huobi.com/
+    img: huobi.svg
+    tfa:
```

```suggestion
    tfa:
      - email
```
See https://huobiglobal.zendesk.com/hc/en-us/articles/360000191302